### PR TITLE
Add a hover and padding overrides on Mat List

### PR DESCRIPTION
**Pull request by gaetanmaisse**
**Issue**

NA

**Description**

Add some material overrides for Mat List:
 - When a Mat List is inside a Mat Card then set a negative to balance the default padding of the Mat Card
 - Change the background colour of each list item on hover

### CODE REVIEW

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

A prerelease version of this package has been published! 🚀

To use it in your project run the following lines:
```shell
npm install @gravitee/ui-particles-angular@1.2.1-add-mat-list-override-85ca928
# or
yarn add @gravitee/ui-particles-angular@1.2.1-add-mat-list-override-85ca928
```

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

📚 View the storybook of this branch [here](https://6183b02d73381a003a3be1a6-jkxfgjshjm.chromatic.com)

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

A prerelease version of this package has been published! 🚀

To use it in your project run the following lines:
```shell
npm install @gravitee/ui-particles-angular@1.2.1-add-mat-list-override-1784e69
# or
yarn add @gravitee/ui-particles-angular@1.2.1-add-mat-list-override-1784e69
```

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

📚 View the storybook of this branch [here](https://6183b02d73381a003a3be1a6-kemeodurjj.chromatic.com)

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

A prerelease version of this package has been published! 🚀

To use it in your project run the following lines:
```shell
npm install @gravitee/ui-particles-angular@1.2.1-add-mat-list-override-9f4587a
# or
yarn add @gravitee/ui-particles-angular@1.2.1-add-mat-list-override-9f4587a
```

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

📚 View the storybook of this branch [here](https://6183b02d73381a003a3be1a6-hysihvpscv.chromatic.com)

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

A prerelease version of this package has been published! 🚀

To use it in your project run the following lines:
```shell
npm install @gravitee/ui-particles-angular@1.2.1-add-mat-list-override-ad909c9
# or
yarn add @gravitee/ui-particles-angular@1.2.1-add-mat-list-override-ad909c9
```

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

📚 View the storybook of this branch [here](https://6183b02d73381a003a3be1a6-fpprlhxaip.chromatic.com)

File: ui-particles-angular/projects/ui-particles-angular/src/scss/component/gio-badge/gio-badge.scss [hunk 1]
```diff
@@ -31,8 +31,9 @@
     min-width: 10px;
     padding: 4px 8px;
     font-size: mat.font-size(theme.$mat-typography, caption);
-    font-weight: bold;
+    font-weight: 400;
```
Comment (ThibaudAV) [inline ui-particles-angular/projects/ui-particles-angular/src/scss/component/gio-badge/gio-badge.scss:34]: maybe : `mat.font-weight(theme.$mat-typography, caption);`

**Review by ThibaudAV — APPROVED** [review]

File: ui-particles-angular/projects/ui-particles-angular/src/scss/component/gio-badge/gio-badge.scss (hunk 1 above)
Comment (gaetanmaisse) [reply ui-particles-angular/projects/ui-particles-angular/src/scss/component/gio-badge/gio-badge.scss]: Nice catch!

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

A prerelease version of this package has been published! 🚀

To use it in your project run the following lines:
```shell
npm install @gravitee/ui-particles-angular@1.2.1-add-mat-list-override-76f5610
# or
yarn add @gravitee/ui-particles-angular@1.2.1-add-mat-list-override-76f5610
```

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

A prerelease version of this package has been published! 🚀

To use it in your project run the following lines:
```shell
npm install @gravitee/ui-particles-angular@1.2.1-add-mat-list-override-e76bf98
# or
yarn add @gravitee/ui-particles-angular@1.2.1-add-mat-list-override-e76bf98
```

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

📚 View the storybook of this branch [here](https://6183b02d73381a003a3be1a6-puyyqdlsgz.chromatic.com)

**Pull request merged**